.github/workflows/model_card_consistency_reminder.yml

```
@@ -0,0 +1,39 @@
+name: Model and Dataset Card consistency reminder
+
+on:
+  pull_request:
```

```suggestion
on:
  push: // temporary trigger for testing purposes
  pull_request:
```

>   (note: I did not test this PR. Not sure we can easily test it.)

I think we can test it this way in this PR and if it work, remove the `push` trigger